README.md

```diff
@@ -30,7 +30,7 @@ The voting records
 Each formal vote of the OpenSSL Technical Comittee (OTC) is in the [votes](./votes)
 subdirectory in a separate file.
 
-The voting records are named `yyyymmdd-vote-name.txt` where the `yyyymmdd` is the
+The voting records are named `yyyy-mm-dd-vote-name.txt` where the `yyyy-mm-dd` is the
```

Comment:
The currently used format is actually `vote-yyyy-mm-dd-vote-name.txt`. I‘ll correct that as soon as I am back on my computer.